openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -166,6 +166,11 @@ jQuery(function () {
             .then((module) => module.initCoversSaved());
     }
 
+    if ($('a:contains(People)').length) {
```

Comment:
Is there a better (efficient, robust) selector or selectors we can use then this one? given our current push for i18n this is going to break on other languages.

I'd suggest checking for IDs (adding some if possible).

Reply:
Added an id attribute to the div of class "superNav" under admin/menu